tsfresh/__init__.py

```diff
@@ -18,6 +18,18 @@
     __version__ = 'unknown'
 
 
+# Set default logging handler to avoid "No handler found" warnings.
+import logging
+try:  # Python 2.7+
+    from logging import NullHandler
```

Comment:
This seem a bit heavy to me. 
Can we not just write out a short warning and make a proper documentation of how to use the logging? Also I would then add a logging config to all examples. So, if the user starts from scratch by looking at the examples he will know what to do.